README.md

```diff
@@ -101,6 +101,33 @@ In the GNU C Library the rounding mode can be controlled via the [`fesetround` a
 
 If strict compliance is sought with WebAssembly, then the user of Fizzy must ensure to keep the default rounding mode.
 
+
+## Development
+
+### Performance testing
+
+We use the following build configuration for performance testing:
+
+- Release build type (`-DCMAKE_BUILD_TYPE=Release` which enables `-O3` and disables asserts)
+
+- Latest version of GCC or Clang compiler
+
+  The Fizzy built with recent versions of GCC and Clang has comparable performance.
+  For LTO builds of Fizzy 0.4.0 Clang 10/11 gives -4% better results than GCC 10.
```

Comment:
`-4% better results` is confusing, if it's faster, then `4% better`, or `+4% better` or `+4% faster`
```suggestion
  For LTO builds of Fizzy 0.4.0 Clang 10/11 gives 4% better results than GCC 10.
```

Reply:
I prefer consistently using negative values meaning runtime decreased. So I guess we should re-word the rest of the sentence.

Reply:
Please check it now.